libraries/lib-files/FileNames.h

```diff
@@ -109,12 +109,28 @@ namespace FileNames
       FilePaths &otherNames, wxFileName &newName);
 
    FILES_API wxString LowerCaseAppNameInPath( const wxString & dirIn);
+
+   /** \brief Audacity user cache directory
+    *
+    * Where audacity keeps its cache squirreled away, by default ~/.cache/audacity/
+    * on Unix, Application Data/Audacity on windows system */
+   FILES_API FilePath CacheDir();
```

Comment:
Let's make all 4 functions to be const!

Reply:
What do you mean? This is a namespace, not a class.

Reply:
Oh shoot, ignore me than